nginx-conf/nginx.conf

```diff
@@ -131,11 +131,6 @@ http {
         }
 
         location ~* \.(js|css|png|jpg|jpeg|gif|ico|svg)$ {
-            # If we're outside of govuk then request no indexing, on the assumption that this is a non-production environment
```

Comment:
Should this be in its own commit?

Reply:
This is part of the fix, this if prevents the try_files working for images that don't exist in the FS so prevents the php controller kicking in. We did comment it out in the gov one instead.